app/views/admin/pages/_content_fields.html.erb

```diff
@@ -3,7 +3,11 @@
 <%= content_tag :div, class: 'nested-fields', data: { controller: 'content', new_record: is_new_record } do %>
   <%= form.input :title %>
   <%= form.input :content_type %>
-  <%= form.input :text, as: :trix %>
+  <li class="input">
```

Comment:
I think I can propose some improvement here.

To be able to do something like this

```
form.input :text, as: trix, embed_youtube: true
```

With deafult option to `false`, to not enable this to all trix editor instances across the app.

You can extend TrixInput, it is in `app/inputs/trix_input.rb` and add a controller there if option is present. I think those extra options could be in `options` hash. Take a look at some examples of extending formtastic inputs either in this repo or in active admin, or just on formtastic docs.